bin/suitest

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
```

Comment:
This line really need?

Reply:
Yes, at least on mac .bin folder contains plain js files and we need it to make it executable not sure about windows but it may also use it to create appropriate .cmd file